airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/activities/JobCreationAndStatusUpdateActivityImpl.java

```diff
@@ -52,6 +53,8 @@
 @Slf4j
 public class JobCreationAndStatusUpdateActivityImpl implements JobCreationAndStatusUpdateActivity {
 
+  private static final String RELEASE_STAGE_ATTRIBUTE_KEY = "release_stage";
```

Comment:
nit: i would think about moving this to https://github.com/airbytehq/airbyte/blob/master/airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricTags.java